plugins/workbench-resources/src/components/Workbench.svelte

```diff
@@ -184,11 +187,14 @@
   async function updateWindowTitle (loc: Location) {
     const ws = loc.path[1]
     const docTitle = await getWindowTitle(loc)
+    const unreadCountTitle = newNotificationsCount > 0 ? '(' + newNotificationsCount + ') ' : ''
```

Comment:
We probably don't want to show the exact number if it exceeds some value. E.g. 99 max and then 99+ for the rest.

Reply:
fixed